lib/graphiti/runner.rb

```diff
@@ -1,18 +1,27 @@
 module Graphiti
   class Runner
     attr_reader :params
-    include Graphiti::Base
```

Comment:
Agree moving from `Base` to `Runner` is a good move. Just to be clear, you moved things but did not refactor correct?

Reply:
Correct